tests/console/firewalld.pm

```diff
@@ -0,0 +1,142 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test FirewallD basic usage
+# Maintainer: Alexandre Makoto Tanno <atanno@suse.com>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use transactional_system 'trup_install';
```

Comment:
`use transactional_system 'trup_install';` why is this here ?

Reply:
Hmmm, this came from another test I used as a template. I will remove this.

Reply:
Removed 'use transactional_system 'trup_install'' ' from the test.